### PR TITLE
Remove all hard-coded values

### PR DESCRIPTION
**Pull request by rogerhutchings**
Move hard-coded values to `config.json`, and provide instructions on what the config file should include.

Also includes some code changes based on available ES6 features in Node Argon.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,85 +1,79 @@
...
+const firebase = require('firebase');
+const fetch = require('isomorphic-fetch');
+const config = require('./config.json');
 
 // Constants
 const PORT = 8080;
-const productionUrl = 'https://www.zooniverse.org/api';
-const stagingUrl = 'https://panoptes-staging.zooniverse.org/api';
-const API_URL = (process.env.NODE_ENV === 'production') ? productionUrl : stagingUrl;
+const PRODUCTION_URL = 'https://www.zooniverse.org/api';
```
Comment (simoneduca) [inline app.js:7]: Is there a naming convention I am missing that recommends capitals?

Comment (rogerhutchings) [reply app.js:7]: Constant primitives like strings. Have a look in the `panoptes-javascript-client` config file.

Comment (simoneduca) [reply app.js:7]: Cool. Btw I thought ES6 wasn't supported, so cheers for proving that's wrong :)

Comment (rogerhutchings) [reply app.js:7]: It's not everything in ES6 - http://node.green/

**Comment by rogerhutchings** [comment]
Okay, this should be ready to go, and is now documented. Note that instead of requiring in the firebase credentials, we can just pass the constructor a path string to them - https://firebase.google.com/docs/server/setup#initialize_the_sdk

**Review by simoneduca — COMMENTED** [review]
LGMT. We will need to change the set up files on in the infrastructure, I assume, because atm `config.json` contains the service account details.

**Pull request merged**